src/GraphControl/Control/Grapher.cpp

```diff
@@ -433,6 +436,108 @@ namespace GraphControl
+                    {
+                        m_solver->FormatOptions().SetFormatType(FormatType::MathML);
+                        if (S_OK
+                            == analyzer->PerformFunctionAnalysis(
+                                (Graphing::Analyzer::NativeAnalysisType)Graphing::Analyzer::PerformAnalysisType::PerformAnalysisType_All))
+                        {
+                            Graphing::IGraphFunctionAnalysisData functionAnalysisData = m_solver->Analyze(analyzer.get());
+                            {
+                                equation->XIntercept = ref new String(functionAnalysisData.Zeros.c_str());
+                                equation->YIntercept = ref new String(functionAnalysisData.YIntercept.c_str());
+                                equation->Domain = ref new String(functionAnalysisData.Domain.c_str());
+                                equation->Range = ref new String(functionAnalysisData.Range.c_str());
+                                equation->Parity = functionAnalysisData.Parity;
+
+                                // There is only ever one item in Periodicity. Map is being used because it works with DependencyProperty.
+                                for (auto p : functionAnalysisData.Periodicity)
+                                {
+                                    equation->Periodicity->Insert(p.first.ToString(), ref new String(p.second.c_str()));
+                                }
+
+                                equation->Minima = ConvertWStringVector(functionAnalysisData.Minima);
+                                equation->Maxima = ConvertWStringVector(functionAnalysisData.Maxima);
+                                equation->InflectionPoints = ConvertWStringVector(functionAnalysisData.InflectionPoints);
+                                equation->Monotonicity = ConvertWStringIntMap(functionAnalysisData.MonotoneIntervals);
+                                equation->VerticalAsymptotes = ConvertWStringVector(functionAnalysisData.VerticalAsymptotes);
+                                equation->HorizontalAsymptotes = ConvertWStringVector(functionAnalysisData.HorizontalAsymptotes);
+                                equation->ObliqueAsymptotes = ConvertWStringVector(functionAnalysisData.ObliqueAsymptotes);
+                                equation->TooComplexFeatures = functionAnalysisData.TooComplexFeatures;
+                                continue;
+                            }
+                        }
+                        else
+                        {
+                            equation->AnalysisNotSupported = true;
+                        }
+                    }
+                    else
+                    {
+                        equation->AnalysisNotSupported = true;
+                    }
+                }
+            }
+        }
+    }
+
+    Windows::Foundation::Collections::IObservableVector<Platform::String ^> ^ Grapher::ConvertWStringVector(vector<wstring> inVector)
```

Comment:
You can remove namespaces in the cpp files to make it cleaner, here and below/above